### PR TITLE
Implement pagination logic for slider

### PR DESCRIPTION
**Pull request by sam-warren**
# Fixes or Implements [AB#9205](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9205)

##NOTE: ALSO CONTAINS CHANGES FROM 9111. MERGE 9111 BEFORE REVIEWING THIS.

* [X] Enhancement
* [ ] Bug
* [ ] Documentation

## Description

Implemented pagination logic to set entries per page based on user selection in filter modal.

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [X] Not Required

### UI Changes

NO

### Browsers Tested

* [X] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox
  
## Reviewer Guidelines

A reviewer should examine the PR if the Status above indicates the PR is **Ready**.
Once ready, the reviewer should read the information below as well as supporting information like the Azure ticket to ensure they have a good understanding of the request.

The reviewer should then:

1. Is the PR template complete and accurate?

    Confirm by reading all sections below.

1. Confirm the merge branch

    A normal feature/bug branch should merge into the Dev branch.  Exceptions for hotfix and sprint merges do occur and the target would then be Master.

1. Do the code commits have an appropriate comment and include linkage to the azure ticket by including a reference to "AB#nnnn"?

1. Review the Code Changes

    1. Do the changes match the description?

        The code commits should reflect the documented description and nothing more.  If additional changes are included the reviewer should call these out and request that the developer document the changes in Azure and in the description or remove the changed code.

    1. useful tips

        * Any obvious errors or omissions?
        * Does the code match the style of other similar work?
        * Is the code easy to understand?
        * Is the code well documented?
        * DB Migrations are appropriate?

If the Answer to any of the questions above is No, then a request for change should be the result of the review.

### CODE REVIEW

**Review by sslaws — DISMISSED** [review]

**Review by leobcgov — APPROVED** [review]
mbe better with some screenshots for the UI changes

File: Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue [hunk 1]
```diff
@@ -86,6 +90,11 @@ export default class FilterComponent extends Vue {
         this.isVisible = false;
     }
 
+    @Watch("isListView")
+    private onIsListView() {
+        !this.isListView ? (this.showSlider = false) : (this.showSlider = true);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue:95]: this looks odd, how about:
```
this.showSlider =  this.isListView;
```
and if that is the case, why is the variable showSlider needed?

**Review by FuriousLlama — CHANGES_REQUESTED** [review]
looks like this.isListView can be simplified

File: Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue (hunk 1 above)
Comment (sam-warren) [reply Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue:95]: good point! will change this

**Review by FuriousLlama — APPROVED** [review]

**Review by sslaws — APPROVED** [review]

**Review by leobcgov — APPROVED** [review]

**Pull request merged**